R/utilities-simulation-set.R

```diff
@@ -199,10 +199,8 @@ getSimulationTimeRanges <- function(simulation, path, simulationSet) {
   logDebug(messages$numberOfApplications(length(applications), path, simulation$name))
   logDebug(messages$timeRangesForSimulation(paste0(simulationRanges, collapse = "', '"), simulation$name))
 
-  # Define ranges for output
-  # Depending on expected behaviour of settings$applicationRange
-  # It would be possible to set these values
-  timeRanges$total$values <- c(min(simulationRanges), max(simulationRanges))
+  # Define ranges for output based on time offset and simulation
+  timeRanges$total$values <- c(simulationSet$timeOffset, max(simulationRanges))
```

Comment:
I think this is wrong. What if the first application is given not at t=0?
Should be something like `c(min(simulationRanges, simulationSet$timeOffset), max(simulationRanges))`

Reply:
Then, I have a very naive question.
Was the goal of the offset only to start at the time of first application if the application started >0 ? 
If so, is there actually a need for the offset ?

Reply:
Yes, we could produce a warnig, I also could not think of a use case where time offset is not equal to an application start time.